homeassistant/components/smartthings/sensor.py

```diff
@@ -178,25 +256,43 @@
             "Robot Cleaner Cleaning Mode",
             None,
             None,
+            None,
         )
     ],
     Capability.robot_cleaner_movement: [
-        Map(Attribute.robot_cleaner_movement, "Robot Cleaner Movement", None, None)
+        Map(
+            Attribute.robot_cleaner_movement, "Robot Cleaner Movement", None, None, None
+        )
     ],
     Capability.robot_cleaner_turbo_mode: [
-        Map(Attribute.robot_cleaner_turbo_mode, "Robot Cleaner Turbo Mode", None, None)
+        Map(
+            Attribute.robot_cleaner_turbo_mode,
+            "Robot Cleaner Turbo Mode",
+            None,
+            None,
+            None,
+        )
     ],
     Capability.signal_strength: [
-        Map(Attribute.lqi, "LQI Signal Strength", None, None),
-        Map(Attribute.rssi, "RSSI Signal Strength", None, None),
+        Map(Attribute.lqi, "LQI Signal Strength", None, None, STATE_CLASS_MEASUREMENT),
+        Map(
+            Attribute.rssi,
+            "RSSI Signal Strength",
+            None,
```

Comment:
The signal strength is normally measured in dB.

https://github.com/home-assistant/core/blob/05a78537200a5ad01b8c0164ee1fb64c01f5b4b8/homeassistant/const.py#L533